.github/actions/test/action.yml

```diff
@@ -15,7 +14,7 @@ runs:
       shell: bash
     - run: chmod +x ./replay-node/macOS-replay-node
       shell: bash
-    - run: node ${{ github.action_path }}/../../../test/run.js
+    - run: npm run test-e2e -- --shard ${{ env.INPUT_STRIPE }}
```

Comment:
this is a nit, but if we already have the term `stripe` can we keep calling them stripes? I'd prefer not to use two words where one will suffice (or maybe there's actually a difference between a shard and a stripe that I am missing?)

Reply:
This is a jest option, so we can't rename it.

Reply:
Ah, ok, makes sense! 👍